### DISCUSSION TITLE
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by bobevski96** [prisma/prisma#22448] [Q&A] [answered]
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

npm i -D prisma
"postinstall": "prisma generate"

and other stuff I have tryed, still not working, the code in the  schema.prisma is just gray , like commented code

**Comment by ludralph** [comment]
Hi @bobevski96 

Thank you for raising this question.

Did you run the `npx prisma generate` command on the command line?

**Reply by bobevski96** [reply]
yes I did, still the same, I am new to prisma so I may have not understand it, like do I need to conect it with other data base, cos I just  install it and confg it

**Reply by bobevski96** [reply]
![image](https://github.com/prisma/prisma/assets/106536988/2b61e444-92b9-4720-a495-08959ebab532)
and looks like this, like commented code

**Reply by ludralph** [reply, marked as answer]
Can you install the Prisma [VScode extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma)? This will help with syntax highlighting and code formatting for the schema.prisma file. When you run the`npx prisma generate` command you should see something along these lines:
```
raphaeletim@Raphaels-MacBook-Air timescale-prisma % npx prisma generate                        
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma

added 2 packages, and audited 25 packages in 5s

found 0 vulnerabilities

✔ Installed the @prisma/client and prisma packages in your project

✔ Generated Prisma Client (v5.6.0) to ./node_modules/@prisma/client in 46ms

Start using Prisma Client in Node.js (See: https://pris.ly/d/client)
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
or start using Prisma Client at the edge (See: https://pris.ly/d/accelerate)
```
import { PrismaClient } from '@prisma/client/edge'
const prisma = new PrismaClient()
```

See other ways of importing Prisma Client: http://pris.ly/d/importing-client
```
Did you get similar output after running the command?

**Reply by bobevski96** [reply]
I just install that extension  and the problem was gone, dont get it , was just that the problem, took me 5-6 hrs ...

**Reply by janpio** [reply]
The screenshot does not include any models. Without any models, running `prisma generate` should give you an error message with explanation that you need a `model` so it can actually generate anything.

**Comment by bobevski96** [comment]
Ty, after vs extension I installed problem was solves, dont why was such a big deal


Sent from Yahoo Mail for iPhone


On Monday, December 18, 2023, 11:51 AM, Raphael Etim ***@***.***> wrote:




Can you install the Prisma VScode extension? This will help with syntax highlighting and code formatting for the schema.prisma file. When you run thenpx prisma generate command you should see something along these lines:
***@***.*** timescale-prisma % npx prisma generate                        
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma

added 2 packages, and audited 25 packages in 5s

found 0 vulnerabilities

✔ Installed the @prisma/client and prisma packages in your project

✔ Generated Prisma Client (v5.6.0) to .***@***.***/client in 46ms

Start using Prisma Client in Node.js (See: https://pris.ly/d/client)

import { PrismaClient } from ***@***.***/client'
const prisma = new PrismaClient()
or start using Prisma Client at the edge (See: https://pris.ly/d/accelerate)

import { PrismaClient } from ***@***.***/client/edge'
const prisma = new PrismaClient()

See other ways of importing Prisma Client: http://pris.ly/d/importing-client

Did you get similar output after running the command?
